pkg/transformers/namespace.go

```diff
@@ -32,23 +32,24 @@ var namespacePathConfigs = []PathConfig{
 		Path:               []string{"metadata", "namespace"},
 		CreateIfNotPresent: true,
 	},
+	{
+		GroupVersionKind:   &schema.GroupVersionKind{Kind: "ClusterRoleBinding"},
+		Path:               []string{"subjects", "namespace"},
+		CreateIfNotPresent: true,
+		IgnoreIfPresent:    true,
```

Comment:
Turns out this will actually introduce a new bug.  `ClusterRoleBinding` subjects can be cluster scoped (`Group`, `User`) or namespace scoped (`ServiceAccount`), but we'll be adding a namespace to all subjects with the current implementation. We'd need to change the implementation here to be more context aware (could be useful in other cases as well), or go back to the variable substitution workaround. Thoughts?

Reference:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#subject-v1beta1-rbac-authorization-k8s-io